### PR TITLE
[DEV] Changes to scrapper workflow. Tests for score eight and ten for pipeline [skip-name]

### PR DESCRIPTION
**Pull request by dmitry-uraev**

### CODE REVIEW

File: requirements_qa.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-pylint==2.6.0
\ No newline at end of file
+pylint==2.6.0
+numpy==1.20.1
```
Comment (demid5111) [inline requirements_qa.txt]: no, that should not be here. students will have to add these dependencies on their own

File: visualizer.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    axis.set_xticklabels(sorted_tags)
+    plt.setp(sorted_tags)
+    plt.xticks(rotation=20)
+    y_max = max(sorted_frequencies) + 1
+    plt.ylim(0, y_max)
+
+    plt.savefig(path_to_save)
+
+
+if __name__ == "__main__":
```
Comment (demid5111) [inline visualizer.py:36]: remove, we never assume that this module is collable, so we do not need if name == main

**Review by demid5111 — CHANGES_REQUESTED** [review]

File: visualizer.py (hunk 2 above)
Comment (dmitry-uraev) [reply visualizer.py:36]: fixed

File: requirements_qa.txt (hunk 1 above)
Comment (dmitry-uraev) [reply requirements_qa.txt]: fixed

**Pull request merged**